scvi/model/_peakvi.py

```diff
@@ -351,6 +346,7 @@ def differential_accessibility(
         batchid2: Optional[Iterable[str]] = None,
         fdr_target: float = 0.05,
         two_sided: bool = True,
+        silent: bool = False,
```

Comment:
move this underneath fdr target and then revert back the {doc_differential_expression}